### PR TITLE
ch03s02(PTL-11395): intro jupiter

### PR DESCRIPTION
**Pull request by alxlenc**
Related to:
* https://github.com/RedHatTraining/AI262/pull/20

### CODE REVIEW

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 1]
```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "1. Edit the following markdown cell with the *CHANGE_ME* text.\n",
```
Comment (jramcast) [inline intro/notebooks-intro/notebooks-intro.ipynb:9]: The indentation in this step is a bit strange
image

We can fix it if we just add a tab in the sentences under the numbered step.
image

image

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    "Then, press `Enter` to edit the cell.\n",
+    "Alternatively, you can double click the cell.\n",
+    "\n",
+    "To render the cell, press `Ctrl+Enter`.\n",
+    "You can also render the cell and advance to the next cell by pressing `Shift+Enter`.\n",
+    "If no next cell exists, then pressing `Shift+Enter` adds a new cell at the bottom of the notebook.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (jramcast) [inline intro/notebooks-intro/notebooks-intro.ipynb:22]: This cell should be a markdown cell
```suggestion
   "cell_type": "markdown",
```

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+    "To do this, replace the `CHANGE_ME` string in the image name.\n",
+    "\n",
+    "![Red Hat logo](CHANGE_ME.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
```
Comment (jramcast) [inline intro/notebooks-intro/notebooks-intro.ipynb:116]: We don't need this empty cell

```suggestion
```

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+    "from IPython import display\n",
+    "\n",
+    "display.Image(\"CHANGE_ME\", width=600, height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "7. Uncomment the last two lines from the following source code and run the cell to plot a sine graph.\n"
```
Comment (jramcast) [inline intro/notebooks-intro/notebooks-intro.ipynb:142]: There are no commented lines in the following cell.

**Review by jramcast — COMMENTED** [review]
Nice job! I have flagged a few formatting issues. Other than that, no major issues.

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+   "metadata": {},
+   "source": [
+    "13. Create an ordered sequence from 1 to 5 by moving the following cells.\n",
+    "\n",
+    "Click the `+` icon in the toolbar to add the missing element.\n",
+    "\n",
+    "To move a cell, hover over the left margin of a cell until the pointer cursor changes to a move cursor.\n",
+    "Then, drag the cell to the desired position.\n",
+    "\n",
+    "To delete a cell, click the bin icon at the top-right corner of the cell."
```
[the remark this answers is not in the document]
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb]: [7e221ef](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/7e221efb4dbfb059ae5dae4f8b40003867a2259d)

File: intro/notebooks-intro/notebooks-intro.ipynb (hunk 4 above)
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb:142]: Sorry, that was me doing the exercise

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+    "  4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "14. Export this notebook to an HTML format.\n",
+    "To export a notebook, click `File` in the top navigation bar, then `Save and Export Notebook As...` and finally select the HTML output format."
+   ]
```
[the remark this answers is not in the document]
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb:294]: [a2c6e87](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/a2c6e8705eab488a26d3adc619ba98dcc5b6dab9)

File: intro/notebooks-intro/notebooks-intro.ipynb (hunk 3 above)
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb]: [a2c6e87](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/a2c6e8705eab488a26d3adc619ba98dcc5b6dab9)

File: intro/notebooks-intro/notebooks-intro.ipynb (hunk 4 above)
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb:135]: [a2c6e87](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/a2c6e8705eab488a26d3adc619ba98dcc5b6dab9)

File: intro/notebooks-intro/notebooks-intro.ipynb (hunk 2 above)
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb]: [a2c6e87](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/a2c6e8705eab488a26d3adc619ba98dcc5b6dab9)

**Review by jramcast — APPROVED** [review]
+1. Some markdown cells still have the indentation problem and they render in what it seems a smaller font 
image

**Comment by alxlenc** [comment]
I had to remove paragraphs in [fc75cdb](https://github.com/RedHatTraining/AI26X-apps/pull/4/commits/fc75cdbcb2cf3e5353af2d0704c44ab91ba25dae). If you try to use paragraphs in numbered lists JupyterLab changes the text format.

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+    "from IPython import display\n",
+    "\n",
+    "display.Image(\"CHANGE_ME\", width=600, height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "7. Uncomment the last two lines from the following source code and run the cell to plot a sine graph.\n"
```
Comment (sffrench) [inline intro/notebooks-intro/notebooks-intro.ipynb:128]: Excuse my ignorance on this one, I just want to check that you meant to say "sine" graph and not "line" graph. I don't know what a sine graph is and it has close spelling to line, just want to check it is not a typo.

File: intro/notebooks-intro/notebooks-intro.ipynb [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "  4"
+   ]
```
Comment (sffrench) [inline intro/notebooks-intro/notebooks-intro.ipynb:265]: Again excuse my ignorance, this all looks a little different and I don't understand the technology. Is this supposed to appear in the preview as: 
1
3
5
4

**Review by sffrench — COMMENTED** [review]
I think my questions are just from my lack of knowledge rather than edits. Quick review complete.

File: intro/notebooks-intro/notebooks-intro.ipynb (hunk 7 above)
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb:128]: No worries Sam. A sine is a math function that prints a graph like the following.
https://images.app.goo.gl/6Nhi19y9GZ9t6fb2A

File: intro/notebooks-intro/notebooks-intro.ipynb (hunk 8 above)
Comment (alxlenc) [reply intro/notebooks-intro/notebooks-intro.ipynb:265]: Yeap, that is part of the exercise. We present an unordered sequence and the student has to drag the numbers to the correct place.

**Pull request merged**